examples/robot_example.cpp

```diff
@@ -0,0 +1,94 @@
+
+// #include "kodlab_mjbots_sdk/robot_control_loop.h"
+// #include "kodlab_mjbots_sdk/behavior.h"
+// #include "kodlab_mjbots_sdk/robot_super.h"
+#include <vector>
+#include <iostream>
+#include <memory>
+#include "kodlab_mjbots_sdk/common_header.h"
+#include "ManyMotorLog.hpp"
+#include "ModeInput.hpp"
+#include "kodlab_mjbots_sdk/robot_interface.h"
+#include "kodlab_mjbots_sdk/mjbots_control_loop.h"
+
+#include "examples/simple_robot.h"
+
+class SimpleRobotControlLoop : public kodlab::mjbots::MjbotsControlLoop<ManyMotorLog, ModeInput, SimpleRobot>
+{
+    using MjbotsControlLoop::MjbotsControlLoop;
+
+    void CalcTorques() override
+    {
+        robot_->Update();
```

Comment:
Not sure if this is the interaction we want or if this should be hidden in robot unless you want an override

Reply:
I think the latter.  This would permit the override if desired (and support backwards compatability) while encouraging use of custom `Robot` classes, which will provide better framework and added functionality (e.g. multiple behaviors).

Reply:
I think we should not change this. Keep it as: `  virtual void CalcTorques() = 0;`

_Originally posted by @ShaneRozenLevy in https://github.com/KodlabPenn/kodlab_mjbots_sdk/pull/22#discussion_r931545298_

Reply:
I think Shane has a point.  Keep this in `robot_example.cpp` and comment appropriately.  Then make `CalcTorques` (now `Update`) pure virtual in `MjbotsControlLoop`.

Reply:
`CalcTorqes` made pure virtual in 060a657fd0508b266f5dde0c99b0d6b2a72ed7e7. Documentation of this override should be clarified in a robot_example.cpp documentation update commit.

Reply:
LGTM